osu.Server.Spectator.Tests/Multiplayer/MultiplayerAllPlayersRoundRobinQueueTests.cs

```diff
@@ -28,55 +28,30 @@ public async Task RoundRobinOrdering()
             await checkCurrentItem(1);
 
             // User 1 adds an extra item
-            await Hub.AddPlaylistItem(new MultiplayerPlaylistItem
-            {
-                BeatmapID = 3333,
-                BeatmapChecksum = "3333"
-            });
+            await addItem();
 
             await checkCurrentItem(1);
-            await checkOrder(1, 0);
-            await checkOrder(2, 1);
+            await checkOrder(1, 2);
 
             // Item played.
             await runGameplay();
             await checkCurrentItem(2);
-            await checkOrder(2, 0);
+            await checkOrder(2);
 
             // User 2 adds two items.
             SetUserContext(ContextUser2);
-
-            await Hub.AddPlaylistItem(new MultiplayerPlaylistItem
-            {
-                BeatmapID = 3333,
-                BeatmapChecksum = "3333"
-            });
-
-            await Hub.AddPlaylistItem(new MultiplayerPlaylistItem
-            {
-                BeatmapID = 3333,
-                BeatmapChecksum = "3333"
-            });
+            await addItem();
+            await addItem();
 
             await checkCurrentItem(2);
-            await checkOrder(2, 0);
-            await checkOrder(3, 1);
-            await checkOrder(4, 2);
+            await checkOrder(2, 3, 4);
 
             // User 1 adds an item.
             SetUserContext(ContextUser);
-
-            await Hub.AddPlaylistItem(new MultiplayerPlaylistItem
-            {
-                BeatmapID = 3333,
-                BeatmapChecksum = "3333"
-            });
+            await addItem();
 
             await checkCurrentItem(2);
-            await checkOrder(2, 0);
-            await checkOrder(3, 1);
-            await checkOrder(5, 2);
-            await checkOrder(4, 3);
+            await checkOrder(2, 3, 5, 4);
```

Comment:
There's no actual changes to this test method, just extracting the first column here into the params array. The second column is redundant and will always be 0, 1, 2, 3, etc.., so it's been removed.